test/App.spec.js

```diff
@@ -0,0 +1,46 @@
+import { mount } from '@vue/test-utils';
+import app from '../src/App.vue';
+
+describe('App.vue', () => {
+
+  const wrapper = mount(app);
+  it('renders the todo app', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+  });
+
+  //for some unforsaken reason, this one refuses to work.
```

Comment:
Why?

Reply:
works now. before we accessed button (0) which also should have been an edit button. however it never worked there. accessing button(2) worked ((1) is a delete button).